### DISCUSSION TITLE
Ads present

### DISCUSSION THREAD

**Discussion by UltimateSJ** [pear-devs/pear-desktop#2026] [Q&A] [answered]
Hiya,

Really appreciate this application, but I've noticed in the last few days that some ads are no longer being blocked despite the ad blocker being enabled. I have reloaded and reset several times. If anyone knows how to resolve this issue, it would be much appreciated.

Thanks!

**Comment by DaveThuet** [comment]
I have the same Problem...
After two Videos i have ADS...
😔

**Comment by hn-on-fire** [comment]
Yes similar issues are arising for me as well, I am guessing it is because youtube changed the way it integrates ads...

**Comment by PolarisXLL** [comment]
Seems like most adblockers were bricked with whatever update youtube rolled out

**Comment by JellyBrick** [comment, marked as answer]
Duplicate of #1817
